include/usb/usb_device.h

```diff
@@ -44,8 +44,13 @@
 extern "C" {
 #endif
 
+#if defined(CONFIG_LOG_FUNCTION_NAME)
```

Comment:
now that logging function name is supported by the logging subsystem, shouldn't we remove all of this and move from USB_ to LOG_ directly?

Reply:
Yes, I will do this.

Reply:
yes please remove

Reply:
BTW, this is done.